client/styles/settings.less

```diff
@@ -300,6 +301,10 @@ section.mainSection {
 			font-size: 24px;
 			color: @trans-grey-56;
 			.transition01();
+			-webkit-transform-origin: 50% 50%;
+			transform-origin: 50% 50%;
+			-webkit-backface-visibility: hidden;
+			backface-visibility: hidden;
```

Comment:
whoop whoop ... this fixes the misplacing of the rotated icon in opera. Now it works like in chrome :)